docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,21 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, either use EESSI with Lmod:
```

Comment:
I think you need to give people a little more guidance here about when to use which approach. If you are already using Lmod >= X, you can use the first option (and you should consider unsetting `MODULEPATH`). If you use anything other than Lmod you should definitely unset `MODULEPATH` as if Lmod sees it it will try to build a cache for it. If you use Tmod or have no module tool, then you need to initialise Lmod.